modular_skyrat/modules/cargo/code/goodies.dm

```diff
@@ -1,11 +1,69 @@
+//////////////////////////////////////////////////////////////////////////////
+//////////////////////// Emergency Race Stuff ////////////////////////////////
+//////////////////////////////////////////////////////////////////////////////
+
+/datum/supply_pack/goody/airsuppliesnitrogen
+	name = "Emergency Air Supplies (Nitrogen)"
+	desc = "A vox breathing mask and nitrogen tank."
+	cost = PAYCHECK_MEDIUM
+	contains = list(/obj/item/tank/internals/nitrogen/belt,
+                    /obj/item/clothing/mask/breath/vox)
+
+/datum/supply_pack/goody/airsuppliesoxygen
+	name = "Emergency Air Supplies (Oxygen)"
+	desc = "A breathing mask and emergency oxygen tank."
+	cost = PAYCHECK_MEDIUM
+	contains = list(/obj/item/tank/internals/emergency_oxygen,
+                    /obj/item/clothing/mask/breath)
+
+/datum/supply_pack/goody/airsuppliesplasma
+	name = "Emergency Air Supplies (Plasma)"
+	desc = "A breathing mask and plasmaman plasma tank."
+	cost = PAYCHECK_MEDIUM
+	contains = list(/obj/item/tank/internals/plasmaman/belt,
+                    /obj/item/clothing/mask/breath)
+
+//////////////////////////////////////////////////////////////////////////////
+///////////////////////////// Misc Stuff /////////////////////////////////////
+//////////////////////////////////////////////////////////////////////////////
+
+/datum/supply_pack/goody/crayons
+	name = "Box of Crayons"
+	desc = "Colorful!"
+	cost = PAYCHECK_MEDIUM * 2
+	contains = list(/obj/item/storage/crayons)
+
+/datum/supply_pack/goody/diamondring
+	name = "Diamond Ring"
+	desc = "Show them your love is like a diamond: unbreakable and everlasting. No refunds."
+	cost = PAYCHECK_MEDIUM * 50
+	contains = list(/obj/item/storage/fancy/ringbox/diamond)
+	crate_name = "diamond ring crate"
+
+/datum/supply_pack/goody/paperbin
+	name = "Paper Bin"
+	desc = "Pushing paperwork is always easier when you have paper to push!"
+	cost = PAYCHECK_MEDIUM * 4
+	contains = list(/obj/item/paper_bin)
+
+//////////////////////////////////////////////////////////////////////////////
+//////////////////////////// Weapons or Ammo /////////////////////////////////
+//////////////////////////////////////////////////////////////////////////////
+
+/datum/supply_pack/goody/minienergygun
+	name = "Mini Energy Gun"
+	desc = "For when you want a little extra insurance. Not very strong, but a decent deterrent." //20 Damage a hit, maximum of 120 damage before recharge.
```

Comment:
With new combat, not sure how viable this is but it ain't supposed to be a showstopper so *shrug*

Reply:
AFAIK the testmerge for the combat rework is currently down. The entire point of it is that it's powerful enough to be a deterrent, but not enough to win a fight. You know, unless you carry around more than one.

It's enough to -maybe- get an edge in a fight. Or something that a sec officer -might- overlook because it's not really a threat.